frontend/ui/elements/reader_menu_order.lua

```diff
@@ -121,6 +121,7 @@ local order = {
         "news_downloader",
         "send2ebook",
         "text_editor",
+        "move_to_archive",
```

Comment:
I'm inclined to think this would fit better near progress_sync if we insert it in this part of the menu. The more_plugins menu will be pushed down no matter where above it you insert an extra entry.

Pinging @NiLuJe @poire-z @pazos for some thoughts.

Reply:
I guess the more_plugins menu isn't really needed anymore, now that the user can disable the plugins he doesn't need.
If you put some in more_plugins, no matter the user cleaning, he would have to tap once more to reach that unfortunate plugin we put into more_plugins.
If we'd want to keep more_plugins, only non-user-action plugins (those with only settings or some permanent behaviour) should go in there.

Reply:
Yes, I have only a few plugins activated, and definitely do not need More plugins menu.

Reply:
I agree that the more plugins menu in not necessary, and just annoying for the plugins that i use that are there. (I only use ~5 plugins)